rts/Sim/Units/UnitHandler.cpp

```diff
@@ -408,13 +412,15 @@ void CUnitHandler::UpdateUnits()
 {
 	SCOPED_TIMER("Sim::Unit::Update");
 
+	maxUnitAltitude = 0.0f;
```

Comment:
Does the 0 refer to the water surface or is just a "low" number? Consider picking one of the below accordingly:
```suggestion
	maxUnitAltitude = CGround::GetWaterPlaneLevel();
	maxUnitAltitude = std::numeric_limits<float>::lowest();
```

Reply:
Ok changed this to lowest on Kill(), but GroundMaxHeight otherwise since that's the lowest we should start testing from.